docs/modules/user-guide/partials/proc_defining-endpoints.adoc

```diff
@@ -2,7 +2,11 @@
 = Defining endpoints
 
 [role="_abstract"]
-Components of any type can specify the endpoints that the container exposes. These endpoints can be made accessible to the users if the {prod-short} cluster is running using a Kubernetes ingress or an OpenShift route and to the other components within the workspace. You can create an endpoint for your application or database, if your application or database server is listening on a port and you want to be able to directly interact with it yourself or you want other components to interact with it.
+Use any type of components to specify the endpoints that the container exposes. You can access these endpoints if the devfile cluster runs by using a Kubernetes ingress or an OpenShift route. You can also connect these endpoints to the other components within the workspace. Creating an endpoint for your application or database helps you:
+
+* If your application or database server listens on a port
+* You want to interact directly with your application or database
+* You want other components to interact with your application or database
```

Comment:
See IBM SG: 
Lead-in wording
Generally, use a sentence to introduce a list that is not a procedure. For guidelines about how to introduce a
procedure, see Procedures.